sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java

```diff
@@ -4,36 +4,36 @@
 import android.os.Looper;
 import android.provider.Settings;
 import android.view.MotionEvent;
-
 import com.facebook.react.bridge.Dynamic;
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.common.SystemClock;
 import com.ooyala.android.OoyalaPlayer;
 import com.ooyala.android.discovery.DiscoveryManager;
+import com.ooyala.android.skin.button.SkinButton;
 import com.ooyala.android.util.DebugMode;
 
+import java.util.HashSet;
+import java.util.Set;
+
 import static com.facebook.react.bridge.UiThreadUtil.runOnUiThread;
 
 /**
  * This class handles all of the events that can come from the React Bridge, and performs all of the necessary actions
  */
 class OoyalaSkinBridgeEventHandlerImpl implements BridgeEventHandler {
   private static String TAG = OoyalaSkinBridgeEventHandlerImpl.class.getSimpleName();
-  private static final String BUTTON_PLAYPAUSE = "PlayPause";
-  private static final String BUTTON_PLAY = "Play";
-  private static final String BUTTON_SHARE = "Share";
-  private static final String BUTTON_REWIND = "rewind";
-  private static final String BUTTON_SOCIALSHARE = "SocialShare";
-  private static final String BUTTON_FULLSCREEN = "Fullscreen";
-  private static final String BUTTON_LEARNMORE = "LearnMore";
-  private static final String BUTTON_MORE_OPTION = "More";
-  private static final String BUTTON_UPNEXT_DISMISS = "upNextDismiss";
-  private static final String BUTTON_UPNEXT_CLICK = "upNextClick";
-  private static final String BUTTON_SKIP = "Skip";
-  private static final String BUTTON_ADICON = "Icon";
-  private static final String BUTTON_ADOVERLAY = "Overlay";
-  private static final String BUTTON_STEREOSCOPIC = "stereoscopic";
-  private static final String PLAYBACK_SPEED_RATE = "playbackSpeedRate";
+
+    public static final Set<SkinButton> SUPPORTED_BUTTONS;
```

Comment:
Are these buttons supported audio buttons?
Maybe incapsulate this set in SkinButton class too, because now it's confusing why only part of buttons are here.

Reply:
Done